frontend/test/metabase/scenarios/question/native.cy.spec.js

```diff
@@ -374,4 +376,90 @@ describe("scenarios > question > native", () => {
+    it.skip(`${test.toUpperCase()} version:\n should be able to view SQL question when accessing via dashboard with filters connected to modified card without SQL permissions (metabase#15163)`, () => {
+      cy.server();
+      cy.route("POST", "/api/dataset").as("dataset");
+
+      cy.signInAsAdmin();
+      cy.createNativeQuestion({
+        name: "15163",
+        native: {
+          query: 'SELECT COUNT(*) FROM "PRODUCTS" WHERE {{cat}}',
+          "template-tags": {
+            cat: {
+              id: "dd7f3e66-b659-7d1c-87b3-ab627317581c",
+              name: "cat",
+              "display-name": "Cat",
+              type: "dimension",
+              dimension: ["field-id", PRODUCTS.CATEGORY],
+              "widget-type": "category",
+              default: null,
+            },
+          },
+        },
+      }).then(({ body: { id: QUESTION_ID } }) => {
+        cy.createDashboard("15163D").then(({ body: { id: DASHBOARD_ID } }) => {
+          // Add filter to the dashboard
+          cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+            parameters: [
+              {
+                name: "Category",
+                slug: "category",
+                id: "fd723065",
+                type: "category",
+              },
+            ],
+          });
+
+          // Add previously created question to the dashboard
+          cy.request("POST", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+            cardId: QUESTION_ID,
+          }).then(({ body: { id: DASH_CARD_ID } }) => {
+            // Connect filter to that question
+            cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+              cards: [
+                {
+                  id: DASH_CARD_ID,
+                  card_id: QUESTION_ID,
+                  row: 0,
+                  col: 0,
+                  sizeX: 10,
+                  sizeY: 8,
+                  series: [],
+                  visualization_settings: {
+                    "card.title": "New Title",
+                  },
+                  parameter_mappings: [
+                    {
+                      parameter_id: "fd723065",
+                      card_id: QUESTION_ID,
+                      target: ["dimension", ["template-tag", "cat"]],
+                    },
+                  ],
+                },
+              ],
+            });
+          });
+
+          if (test === "nosql") {
+            cy.updatePermissionsGraph({
+              [COLLECTION_GROUP]: { "1": { schemas: "all", native: "none" } },
+            });
+          }
+
+          cy.signIn("nodata");
+
+          // Visit dashboard and set the filter through URL
+          cy.visit(`/dashboard/${DASHBOARD_ID}?category=Gizmo`);
+          cy.findByText("New Title").click();
+          cy.wait("@dataset", { timeout: 5000 }).then(xhr => {
+            expect(xhr.response.body.error).not.to.exist;
+          });
+          cy.findByText("Showing 1 row");
```

Comment:
Would this work to make sure editor is not displayed?
```suggestion
          cy.get(".ace_content").should("not.be.visible");
          cy.findByText("Showing 1 row");
```

Reply:
For sure.
I used slightly different assertion - making sure the element doesn't exist. In certain cases it can really be not visible so let's make sure it's not there at all.